### PR TITLE
Updating User model to have more personal fields

### PR DESCRIPTION
**Pull request by nalnaji**
Basically just added some attributes we discussed about on Sunday.

Feel free to rip me apart.

@DarryQueen @bwasmith  @nishadsingh1

### CODE REVIEW

File: db/migrate/20150312234511_add_personal_atts_to_user.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+class AddPersonalAttsToUser < ActiveRecord::Migration
+  def up
+    add_column :users, :name, :string
+    add_column :users, :profile_pic_url, :string
```
Comment (DarryQueen) [inline db/migrate/20150312234511_add_personal_atts_to_user.rb]: Hold on on doing this until we've actually figured out how this works.

File: db/migrate/20150312234511_add_personal_atts_to_user.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    add_column :users, :language, :string
```
Comment (DarryQueen) [inline db/migrate/20150312234511_add_personal_atts_to_user.rb]: Is this necessary? Or even feasible?

Comment (nalnaji) [inline db/migrate/20150312234511_add_personal_atts_to_user.rb]: HAHAHAHA. Shum suggested it. Personally thought it was a ridiculously ambitious request.

Comment (DarryQueen) [inline db/migrate/20150312234511_add_personal_atts_to_user.rb]: A lot of these are cute but purely cosmetic. I'm not opposed to removing all of them, but language just doesn't have any point.

File: db/schema.rb [hunk 3]
```diff
@@ -59,8 +63,8 @@
     t.string   "youtube_id"
     t.string   "title"
     t.text     "description"
-    t.datetime "created_at",   :null => false
-    t.datetime "updated_at",   :null => false
+    t.datetime "created_at",  :null => false
+    t.datetime "updated_at",  :null => false
```
Comment (nalnaji) [inline db/schema.rb]: For some reason the spacing here was messed up. This is the correct spacing as generated by db:migrate. The previous one was wrong.

File: db/schema.rb [hunk 4]
```diff
@@ -50,6 +50,10 @@
     t.datetime "created_at",                             :null => false
     t.datetime "updated_at",                             :null => false
     t.string   "role"
+    t.string   "name"
+    t.string   "city"
+    t.string   "country"
```
Comment (DarryQueen) [inline db/schema.rb]: These two don't have much utility either, but I guess they're kind of cute. If we're keeping this we can find a nice jQuery plugin for autocompletion.

Comment (nalnaji) [inline db/schema.rb]: Let's keep it. It's not a lot, and it would be cool to see where people are using the app from anyway.

Comment (DarryQueen) [inline db/schema.rb]: Okay.

**Comment by DarryQueen** [comment]
Squash.

**Comment by DarryQueen** [comment]
LGTM.

**Pull request merged**